src/backend/storeManagers/hyperplay/library.ts

```diff
@@ -46,6 +47,38 @@ export async function addGameToLibrary(projectId: string) {
   hpLibraryStore.set('games', [...currentLibrary, gameInfo])
 }
 
+// hit the valist api and get the info if the game is a epic listing
+export async function getEpicListingUrl(projectId: string): Promise<string> {
```

Comment:
I think this should be in `src\backend\storeManagers\hyperplay\utils.ts` since we try to keep the store manager library and games modules' api's 1:1 with all the other stores

Reply:
done